### PR TITLE
Feature/2514 rebrand blui

### PR DESCRIPTION
**Pull request by ektasawant**
Fixes 2514 .

#### Changes proposed in this Pull Request:
- [X] @pxblue => @brightlayer-ui

- [X] PX Blue => Brightlayer UI

#### To Test:
- Copy dist folder of login-workflow in `login-workflow/example/node_modules/@brightlayer-ui/react-auth-workflow`.
- Run yarn start:example

### CODE REVIEW

**Comment by emclaug2** [comment]
Can you remove this dep from the example package.json?

`"@brightlayer-ui/react-auth-workflow": "^2.4.0-beta.0"`

When running `yarn start:example`, it builds & copies the dist folder into the example's node_modules so it always runs using the local copy.

**Comment by emclaug2** [comment]
Can you review the docs folder and prune the pxb references?  Check out the imports in the code examples.

**Comment by ektasawant** [comment]
> Can you review the docs folder and prune the pxb references? Check out the imports in the code examples.

Done please review.

File: login-workflow/example/package.json [hunk 1]
```diff
@@ -5,11 +5,11 @@
...
         "@material-ui/core": "^4.11.4",
         "@material-ui/icons": "^4.11.2",
-        "@pxblue/colors": "^3.0.1",
-        "@pxblue/icons-mui": "^2.3.0",
-        "@pxblue/react-auth-workflow": "^2.1.0",
-        "@pxblue/react-components": "^5.2.0",
-        "@pxblue/react-themes": "^6.0.0",
+        "@brightlayer-ui/colors": "^3.0.1",
+        "@brightlayer-ui/icons-mui": "^2.3.0",
+        "@brightlayer-ui/react-auth-workflow": "^2.4.0-beta.0",
```
Comment (emclaug2) [inline login-workflow/example/package.json:10]: This dep is not required.

Comment (emclaug2) [reply login-workflow/example/package.json:10]: This dep needs to be removed:

`"@brightlayer-ui/react-auth-workflow": "^2.4.0-beta.0",`

Our circle ci builds do not test the demo project & it should not require a manual copy/pasting of the dist folder to get this to run.

Comment (ektasawant) [reply login-workflow/example/package.json]: Removed please check.

**Review by emclaug2 — APPROVED** [review]

**Comment by emclaug2** [comment]
Ok cool, let's publish a beta!

**Pull request merged**